code/web/src/modules/survey/Survey.js

```diff
@@ -32,20 +33,44 @@ class Survey extends PureComponent {
         // when survey is submitted, results are displayed
         // on results page, retake quiz or complete subscription
         
-        handleClick = () => {
+        handleNext = () => {
           this.setState({
             isLoading: true
           })
           this.props.nextPage(this.props.survey.page)
         }
+
+        handlePrev = () => {
+          this.setState({
+            isLoading: true
+          })
+          this.props.previousPage(this.props.survey.page)
+        }
```

Comment:
As much as I want to say we should make some code reusable and merge this and next, lets just leave this for now. I love that it's working and don't want to break anything.